### DISCUSSION TITLE
Elipsis på ContextualMenu

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#3641] [Annet]
Er det mulig å få variants av elipsis i komponent?
Fordi dette er skisse:
<img width="212" alt="Screenshot 2023-08-10 at 15 35 45" src="https://github.com/fremtind/jokul/assets/93194579/85456df7-785b-43a1-8d55-17d78dbd3715">
Og dette ble ferdigutviklet (elipsis er "bold"):
<img width="265" alt="Screenshot 2023-08-10 at 15 36 49" src="https://github.com/fremtind/jokul/assets/93194579/3dbc1886-2236-463c-ab48-7c47f850dfb5">

**Comment by leon-ho** [comment]
Man kan velge ikon-tykkelse i koden :)
